paper.tex

```diff
@@ -126,110 +130,134 @@ \section*{Introduction}
+is its flexibility, and numerous elaborations have been
+incorporated~\citep{hudson1990gene,hein2004gene,wakely2008coalescent}.
+In the first extension to the basic process, \cite{hudson1983properties}
+analysed the effects of recombination on the random genealogical
+trees generated by the coalescent,
+producing several fundamental analytical results
+as well as describing the elegant algorithm
+at the heart of simulation
+programs~\citep{hudson2002generating,kelleher2016efficient,baumdicker2021efficient}
+used in thousands of studies.
+In its original formulation, Hudson's algorithm produced a
+sequence of genealogical trees along the genome.
+Critically, although the correlation structure among these
+local trees was analysed analytically, those correlations
+were not captured explicitly: the ancestry of the sample
+was viewed as a series of disconnected local trees rather than
+as a single cohesive object.
+
+In the 1990s, Griffiths and colleagues revisited the
+coalescent with recombination from a different perspective,
+in which the stochastic process itself is encoded as a graph
+(with particular properties; see section XXX for details),
+and showed how the local trees generated by Hudson's algorithm
+were embedded within
+it~\citep{griffiths1991two,ethier1990two,griffiths1996ancestral,griffiths1997ancestral}.
+They referred to this graph,
+representing both the stochastic process and
+its random realisations, as the Ancestral Recombination Graph (ARG).
+Although mathematically equivalent, it is
+important to note that the Griffiths and Hudson formulations of
+the coalescent with recombination are not identical;
+in particular, a direct implementation of the ARG process
+requires exponential time to simulate
+(see Appendix XXX for details). However, ARGs provided a way
+to describe recombinant ancestry as single object that
+could be inferred in a way
+that was missing from Hudson's treatments.
+
+Early work on ARG inference focused on the problem of
+inferring parameters of the
+stochastic process, where the ancestry is regarded as a
+latent parameter to be averaged out
+\citep[e.g.][]{griffiths1996ancestral,kuhner2000maximum, nielsen2000estimation,
+fearnhead2001estimating}.
+These methods met with limited success
+because the state space of ARGs is overwhelmingly large and
+lacks a simple geometry or neighbourhood structure for inference or
+sampling methods to  exploit.
+The focus subsequently shifted to the more
+tractable---but still NP-hard~\citep{wang2001perfect}---problem of computing
+the minimum number of recombinations required
+to explain an input dataset~\citep{myers2003bounds}, and inferring the corresponding
+ARG realisations~\citep{song2003parsimonious,song2005efficient,lyngso2005minimum}.
+The previously close correspondance
+between the ARG as a mathematical representation of the coalescent process
+and as concrete realisations of ancestry was now broken,
+since such parsimonious ARGs are extremely unlikely under the
```

Comment:
I'm not sure it's the value of the likelihood of a parsimonious ARG that is important, it't the fact that it is not constructed WRT to the likelihood itself.

Reply:
Right, but we were now saying that we were interested in ARGs that would be considered neglibly likely under the model, and therefore that the model itself wasn't the thing of primary interest. Maybe this isn't getting across?

Reply:
Perhaps the disctinction to emphasise should just be that the parsimony-based approach ignores branch lengths and focussed on sequences of events / topology? Leading to the concept of the ARG as a data structure.

Reply:
Good point, I forgot about the lack of branch lenghts! That's definitely something to mention. I'll rejig a bit and drop some notes tomorrow.